.github/pull_request_template.md

```diff
@@ -1,32 +1,41 @@
-Thank you for working on TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+<!--
+Thank you for contributing to TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/tidb/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+-->
 
 ## What have you changed? (mandatory)
 
 Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
 - Summarize your change (mandatory)
 - How does this PR work? Need a brief introduction for the changed logic (optional)
-- Describe clearly one logical change and avoid lazy messages (optional)
+- Separately describe each logical change and avoid lazy messages (optional)
 - Describe any limitations of the current code (optional)
 
 Please **NOTE** that:
-- Do not assume the code is self-evident
 - Do not assume reviewers understand the original issue
 
-## What are the type of the changes (mandatory)?
+## Does this PR need to be added in the release note? (mandatory)
 
-The currently defined types are listed below, please pick one of the types for this PR by removing the others:
+If this PR needs to be added in the release note, please add the "**release-note**" for this PR.
```

Comment:
- release note -> release notes
- add the "**release-note**" label?

Reply:
yes, it means the "release-note" label

Reply:
[Here](https://raw.githubusercontent.com/kubernetes/kubernetes/master/.github/PULL_REQUEST_TEMPLATE.md) is the pull request template of K8s.
We could move this section to the last part.